pandas/core/frame.py

```diff
@@ -7909,11 +7909,17 @@ def append(
             to_concat = [self, *other]
         else:
             to_concat = [self, other]
-        return concat(
-            to_concat,
-            ignore_index=ignore_index,
-            verify_integrity=verify_integrity,
-            sort=sort,
+        return (
+            (
+                concat(
+                    to_concat,
+                    ignore_index=ignore_index,
+                    verify_integrity=verify_integrity,
+                    sort=sort,
+                )
+            )
+            .__finalize__(self)
```

Comment:
why are you not calling with method='append' for self?

why are you calling twice? this should only be on self.

Reply:
Removed second call